service/src/main/java/bio/terra/tanagra/service/export/impl/IpynbFileDownload.java

```diff
@@ -32,12 +31,12 @@ public Type getType() {
 
   @Override
   public String getDefaultDisplayName() {
-    return "Download ipynb file with embedded SQL";
+    return "Download Jupyter Notebook";
   }
 
   @Override
   public String getDescription() {
-    return "Signed URL to an ipynb notebook file that includes SQL to get the list of primary entity instances.";
+    return "Get an IPYNB file for download. SQL comes preinstalled.";
```

Comment:
What does it mean to have SQL preinstalled?

Reply:
> What does it mean to have SQL preinstalled?

My assumption is that it includes a SQL kernel and dependencies

Reply:
Do you know if this is true of the existing export implementation?

The generated ipynb file contains the SQL query and the idea is to make it easy to run that query against BigQuery. The template file we use is [here](https://github.com/DataBiosphere/tanagra/blob/main/service/src/main/resources/export/notebook_template.ipynb).

Reply:
@Matthew-Antchak how about 'Get an IPYNB file for download`. From the template - it uses pandas, not SQL kernel.

Reply:
Yes, in that case let's just omit the second sentence about SQL @dexamundsen